letsmeet/users/templates/users/dashboard.html

```diff
@@ -40,8 +40,8 @@
 	{% if request.user %}
         <div class="block-area">
           <div class="well tile">
-	      <a class="btn btn-default" href="{% url 'personal_events_ical_feed' %}"><i class="fa fa-calendar"></i> your personal ical calendar</a>
-            </div>
+	        <a class="btn btn-default" href="{% url 'personal_events_ical_feed' uuid=userprofile.personal_ical_uuid %}"><i class="fa fa-calendar"></i> Your ical calendar</a>
+          </div>
```

Comment:
Indentation is broken, but it was broken before (just in a different way ;) ).
Maybe change it to 4 spaces (to be consistent with the rest of the file).